crates/clvm-traits/docs/derive_macros.md

```diff
@@ -216,3 +295,55 @@ let a = &mut Allocator::new();
 let ptr = value.to_clvm(a).unwrap();
 assert_eq!(Either::from_clvm(a, ptr).unwrap(), value);
 ```
+
+## Hidden Values
+
+Sometimes you may want to include constants inside of a struct without actually exposing them as fields.
+It's possible to do this with `#[clvm(hidden_value)]`, however you must use an attribute macro to remove the hidden fields.
```

Comment:
Yeah, primarily condition opcodes and magic values like the `RunTail` condition CATs use

Reply:
For reference, `RunTail` is `(51 () -113 tail tail_solution)`, and the only non-constant fields are `tail` and `tail_solution`. So it would look like this:

```rs
struct RunTail<P, S> {
    #[clvm(hidden_value = 51)]
    opcode: u8,
    #[clvm(hidden_value = ())]
    puzzle_hash: (),
    #[clvm(hidden_value = -113)]
    magic_amount: i8,
    program: P,
    solution: S,
}
```